frontend/src/components/pages/login/Login.tsx

```diff
@@ -50,6 +57,8 @@ const Login: React.FC = () => {
         </Box>
       </Tile>
       <Footer />
+      {isLoggedIn && history.push('/')}
+      {!isLoggedIn && history.push('/register')}
```

Comment:
why in html? 

please move to the component code part 

should be trinary instead of 2 lines - isLoggedIn ? history.push('/') : history.push('/register')